### PR TITLE
Remove Project.LiveStream

### PR DESCRIPTION
**Pull request by justinswart**
This removes the `Project.LiveStream` sub-struct from `Project` as well as the `liveStreams` property from `Project` as we rather want this to exist separately as `LiveStreamEvent`.

More detail is provided in this PR: https://github.com/kickstarter/ios-oss/pull/63

### CODE REVIEW

File: KsApi/MockService.swift [hunk 1]
```diff
@@ -978,10 +978,10 @@ internal struct MockService: ServiceType {
...
 
   internal func signup(name: String,
-                          email: String,
-                          password: String,
-                          passwordConfirmation: String,
-                          sendNewsletters: Bool) -> SignalProducer<AccessTokenEnvelope, ErrorEnvelope> {
+                       email: String,
+                       password: String,
+                       passwordConfirmation: String,
+                       sendNewsletters: Bool) -> SignalProducer<AccessTokenEnvelope, ErrorEnvelope> {
```
Comment (mbrandonw) [inline KsApi/MockService.swift]: nice, thanks for cleaning these up!

File: KsApi/lib/Route.swift [hunk 2]
```diff
@@ -72,6 +72,7 @@ internal enum Route {
     case video
   }
 
+  // swiftlint:disable:next large_tuple
```
Comment (mbrandonw) [inline KsApi/lib/Route.swift]: can drop this since you globally disabled?

File: KsApi/models/Project.swift [hunk 3]
```diff
@@ -11,7 +11,6 @@ public struct Project {
   public let memberData: MemberData
   public let dates: Dates
   public let id: Int
-  public let liveStreams: [LiveStream]?
```
Comment (mbrandonw) [inline KsApi/models/Project.swift]: sayonara `Project.LiveStream`!

Comment (justinswart) [inline KsApi/models/Project.swift]: 😆

File: KsApi/lib/Route.swift (hunk 2 above)
Comment (justinswart) [inline KsApi/lib/Route.swift]: Oh that global disable is weird, didn't seem to work for ignoring this?

**Pull request merged**